src/XMakeTasks/UnitTests/AssignTargetPath_Tests.cs

```diff
@@ -58,7 +58,9 @@ public void OutOfCone()
                               new TaskItem(
                                   NativeMethodsShared.IsWindows ? @"d:\f1\f2\f3\f4\file.txt" : "/f1/f2/f3/f4/file.txt")
                           };
-            t.RootFolder = NativeMethodsShared.IsWindows ? @"c:\f1" : "/f1";
+            // Note that on Windows we change the drive to make sure the file is out of the cone.
+            // If not Windows, where there is no drive, we change the root directory from f1 and x1
```

Comment:
My concern is a grammar change to the comment itself that might have sped up my realization of that.